pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+_Please tick all applicable boxes, fill in the **bold** fields, and remove these instructions from your PR._
```

Comment:
How about replacing the `_` with `<!--` and `-->`? This way the text won't show when posted (and you don't need the 'remove these instructions from your PR' part).

Also, it's easier for people to tick the boxes after they've posted the PR (they can just click), so maybe change the instructions to: "Please fill in the **bold** fields, submit your pull request and then tick the applicable boxes"?